### PR TITLE
Move perf tests to js/perf/

### PR DESCRIPTION
**Pull request by arv**
This restructures how the perf tests import noms (by doing `require('../../../')` instead of `require('@attic/noms')` which solves the dependency issue.

### CODE REVIEW

**Comment by arv** [comment]
@mikegray @sunglim PTAL

This makes me realize that we should probably just change the whole js structure around to:

```
js/
  noms/  (old js dir)
  perf/
  eslintrc/
  webpack-config/
```

**Comment by mikegray** [comment]
do samples/js then move into js/samples too?

File: js/package.json [hunk 1]
```diff
@@ -36,7 +37,12 @@
     "commander": "^2.9.0",
     "flow-bin": "0.30.0",
     "fs-extra": "^0.30.0",
-    "mocha": "^2.5.3"
+    "humanize": "^0.0.9",
+    "mocha": "^2.5.3",
+    "mz": "^2.4.0",
+    "varint": "^4.0.0",
+    "yargs": "^5.0.0"
+
```
Comment (mikegray) [inline js/package.json]: remove extra blank line?

File: js/perf/codec-perf-rig/src/main.js [hunk 2]
```diff
@@ -43,9 +43,9 @@ const args = argv
   .argv;
 
 main().catch(ex => {
-  console.error('\nError:', ex);
+  console.error('\nError:', ex);  // eslint-disable-line no-console
```
Comment (mikegray) [inline js/perf/codec-perf-rig/src/main.js]: do we still need these with the .eslintrc.js change at root of js/perf/ directory?

**Comment by mikegray** [comment]
lgtm with a few minor nits/questions

File: js/perf/codec-perf-rig/src/main.js (hunk 2 above)
Comment (arv) [inline js/perf/codec-perf-rig/src/main.js]: No. At one point I had the `.eslintrc.js` file removed.

**Comment by arv** [comment]
Lame... that worked locally. I'll just do the big move instead.

**Pull request closed without merging**